test_scripts/API/OpenMenuRPC/commonOpenMenuRPC.lua

```diff
@@ -0,0 +1,91 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+ --[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+
+ --[[ Variables ]]
+local m = actions
+m.type = "FILE"
+
+ --[[ Functions]]
+function m.pTUpdateFunc(pTbl)
+  pTbl.policy_table.functional_groupings["Base-4"].rpcs.ShowAppMenu = nil
+end
+
+function m.addSubMenu(pMenuID)
+  local cid = m.getMobileSession():SendRPC("AddSubMenu",
+    {
+      menuID = pMenuID,
+      position = 500,
+      menuName ="SubMenupositive"
+    })
+    EXPECT_HMICALL("UI.AddSubMenu",

```

Comment:
@TMyza, use common.getHMIConnection():ExpectRequest(...) instead of EXPECT_HMICALL(...)